babel.config.js

```diff
@@ -1,4 +1,22 @@
 module.exports = {
   presets: ['module:@react-native/babel-preset'],
-  plugins: ['react-native-reanimated/plugin'],
+  plugins: [
+    [
+      'module-resolver',
+      {
+        root: ['./src'],
+        extensions: ['.ios.js', '.android.js', '.js', '.ts', '.tsx', '.json'],
+        alias: {
+          '@components': './src/components',
+          '@store': './src/store',
+          '@utils': './src/utils',
+          '@styles': './src/styles',
+          '@mock': './src/mock',
+          '@page': './src/page',
+          '@type': './src/types',
```

Comment:
Seems like very tedious thing to do, every time specify new src sub category here and in tsconfig. Have you considerate using something like `'@': './src'` ?